totpauth-impl/src/main/resources/views/totp-register.vm

```diff
@@ -38,7 +38,8 @@ p.error {
           #end
           <form action="$flowExecutionUrl" method="post">
             #parse("csrf/csrf.vm")
-            <li> Download the Google Authenticator app to your mobile device </li>
+            <li> Download an authenticator application or service to your mobile device that is compatible with RFC 6238. Any application or service which is compatible with RFC 6238 will work. Listed here are some cross device authenticators that are compatible with cloud.gov: Microsoft Authenticator, LastPass, Password Manager, 1password, Authy.
```

Comment:
```suggestion
            <li> Download an authenticator application or service to your mobile device that is compatible with RFC 6238. Any application or service which is compatible with RFC 6238 will work. Listed here are some cross device authenticators that are compatible with cloud.gov: Microsoft Authenticator, LastPass, 1password, Authy.
```

Password Manager was listed but not described below. I also don't know if that's a real thing?